.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       - name: Build a wheel and a sdist
         run: |
-          PYTHONWARNINGS=error,default::DeprecationWarning python -m build --outdir ./dist tests/test_package
+          python -m build --outdir ./dist tests/test_package
```

Comment:
Removing this as I had added in previously from another workflow I have, but we don't care if there are deprecation warnings in the package build tools, as we just need an artifact to upload.

Reply:
that assumes that the command will keep working as is, which I think is a valid assumption (or will be a super simple fix many years down the line), so 👍

Reply:
Yeah, the main reason I removed this is that I've had this check in other work flows and this has ended up causing things to fail for reasons unrelated to `build` itself (e.g. a build backend deprecating something for the future but that warning still getting caught). I agree that if `build` ends up having issues though we will not be the first to catch it. :)